exampleProjects/touch/platformio.ini

```diff
@@ -0,0 +1,31 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[env:tinypico]
+platform = espressif32
+board = tinypico
+framework = arduino
+lib_deps =
+    tinypico/TinyPICO Helper Library
+    adafruit/Adafruit GFX Library
+    adafruit/Adafruit ILI9341
+    adafruit/Adafruit FT6206 Library
+build_flags =
+    -I../../include/
+    -I../../3rdparty/
+    -std=gnu++2a
+    -O0
```

Comment:
I think the -O0 may be contributing to the firmware size issue, why was -Os replaced by it ? to help debugging ?

Reply:
ahh of course!!! Yep, I forgot to comment it out after trying out debugging 🤦